tests/app/celery/test_zip_and_send_letter_pdfs.py

```diff
@@ -87,32 +88,64 @@ def test_zip_and_send_should_update_notifications_to_success(mocks):
     )
 
 
-def test_zip_and_send_should_update_notifications_if_s3_client_error(mocks):
+def test_zip_and_send_should_retry_if_s3_client_error(mocks):
     mocks.get_zip_of_letter_pdfs_from_s3.side_effect = ClientError({}, 'operation')
-
+    mocks.zip_and_send_retry.side_effect = Retry
```

Comment:
Minor: could we make this the default for the mock? (I was a bit surprised that we need to set it, given the name of the mock.)